src/tests/Loader/classloader/InlineArray/InlineArrayValid.cs

```diff
@@ -171,7 +168,7 @@ struct ObjShortArr
         public (object, short) element;
 
         [UnscopedRef]
-        public ref (object o, short s) this[int i] => ref Unsafe.Add(ref element, i);
+        public ref (object o, short s) At(int i) => ref Unsafe.Add(ref element, i);
```

Comment:
I don't think this is necessary.  From #86183 

 >Definition of the element field doesn't include tuple element names, but that is the element type that language is using. To fix, add element names to the declaration of the element field. I think custom indexer can be safely deleted.

So something like this:

```csharp
public (object o, short s) element;
```

and remove `At` and replace `arr.At(i).o` by `arr[i].o` (and same for `.s`) in the uses, below

Reply:
This doesn't seem to work.

```
.../runtime/src/tests/Loader/classloader/InlineArray/InlineArrayValid.cs(178,28): error CS1061: '(object, short)' does not contain a definition for 'o' and no accessible extension method 'o' accepting a first argument of type '(object, short)' could be found (are you missing a using directive or an assembly reference?) [.../runtime/src/tests/Loader/classloader/InlineArray/InlineArrayValid.csproj]
.../runtime/src/tests/Loader/classloader/InlineArray/InlineArrayValid.cs(179,28): error CS1061: '(object, short)' does not contain a definition for 's' and no accessible extension method 's' accepting a first argument of type '(object, short)' could be found (are you missing a using directive or an assembly reference?) [.../runtime/src/tests/Loader/classloader/InlineArray/InlineArrayValid.csproj]
```

```csharp
    [InlineArray(Length)]
    struct ObjShortArr
    {
        public const int Length = 100;
        public (object, short) element;

        // [UnscopedRef]
        // public ref (object o, short s) At(int i) => ref Unsafe.Add(ref element, i);

        [MethodImpl(MethodImplOptions.NoInlining)]
        public static ObjShortArr CreateArray(int recCount) {
            if (recCount > 0) {
                return CreateArray(recCount-1);
            } else {
                var arr = new ObjShortArr();
                for (short i = 0; i < ObjShortArr.Length; i++)
                {
                    arr[i].o = i;
                    arr[i].s = (short)(i + 1);
                }
                return arr;
            }
        }
    }
```

Reply:
I missed the `o` and `s`.